backend/fetchArrivals.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+const key = import.meta.env.VITE_API_KEY;
+
+export default async function getArrivals(id: string){
+    let url = `https://api.tfl.gov.uk/StopPoint/${id}/Arrivals`;
+   
+    if (key !== ""){
```

Comment:
Would it be better to cover scenarios where key is undefined?
Could fo `if (key) {`